use_cases/eluc/.gitignore

```diff
@@ -4,9 +4,11 @@ predictors/*/trained_models/
 # Ignores predictor significance results
 experiments/predictor_significance
 
+experiments/figures
+
 # Ignores trained prescriptors and seeds
-prescriptors/trained_prescriptors
-prescriptors/seeds
+prescriptors/*/trained_prescriptors
+prescriptors/*/seeds
```

Comment:
New project structure with abstract prescriptor class means we need this to look the same as the predictors